src/amber/cli/recipes/file_entries.cr

```diff
@@ -0,0 +1,69 @@
+require "teeplate"
+
+module Amber::Recipes
+  module FileEntries
+    def each_file(abs, rel, &block : String, String ->)
```

Comment:
The names `abs` and `rel` throughout this file are opaque. I'm _guessing_ that they are `absolute_path` and `relative_path`? Similarly `tpl` `d`, etc.

Specifying the full name of a variable is a little more verbose, but it helps the long term maintainability of a codebase.